ci/taos/common/out-of-pr-killer.sh

```diff
@@ -42,19 +43,45 @@ function run_oop_killer(){
     
         # The process killer allows to kill only task(s) in case that there are running lots of tasks with same PR number.
         if [[ ("$victim_pr" -eq "$input_pr") && (1 -eq "$(echo "$victim_date < $input_date" | bc)") ]]; then
-            echo "[DEBUG] victim_pr=$victim_pr, input_pr=$input_pr, victim_date=$victim_date, input_date=$input_date "
-            echo "[DEBUG] killing PR $victim_pr (pid <$victim_pid1> <$victim_pid2> <$victim_pid3>)."
+            echo "[DEBUG] OOP Killer/Tizen: victim_pr=$victim_pr, input_pr=$input_pr, victim_date=$victim_date, input_date=$input_date "
+            echo "[DEBUG] OOP Killer/Tizen: killing PR $victim_pr (pid <$victim_pid1> <$victim_pid2> <$victim_pid3>)."
             kill $victim_pid1
             kill $victim_pid2
             kill $victim_pid3
             sleep 1
             # Handle a possibility that someone updates a single PR multiple times within 1 second.
-            echo "[DEBUG] removing the ./${dir_worker}/${victim_pr}-${victim_date}-* folder"
+            echo "[DEBUG] OOP Killer/Tizen: removing the ./${dir_worker}/${victim_pr}-${victim_date}-* folder"
             rm -rf ./${dir_worker}/${victim_pr}-${victim_date}-*
         fi
     done
 
-    # Todo: NYI, Implement the OOP killer for Ubuntu build (pdebuild)
+    # The OOP killer for Ubuntu build (pdebuild)
+    # Kill PRs that were previously invoked by checker-pr-gateway.sh with the same PR number.
+    echo "[DEBUG] OOP Killer/Ubuntu: Starting killing activity to kill previously invoked checker-pr-gateway.sh with the same PR number.\n"
+    ps aux | grep "^www-data.*bash \./checker-pr-gateway.sh" | while read line
```

Comment:
This point is next of tizen oop, so there may be no '/checker-pr-gateway.sh '

Reply:
Good point.  It is no problem.  The duplicated statements does not effect any issue to system although the Tizen OOP killer executed the same statement beforehand. I intend to consider various use cases to support a portability and compatibility.  For example,  please think about the case that a CI administrator try to enable the Ubuntu build module only to compile their GitHub repository.